mitodl/unified-ecommerce
# Separate baskets/checkout for each integrated system and user

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/5807

### Description (What does it do?)
This updates a bunch of the URLs to make them "integrated system" specific.  It also updates the logic of the basket to make it "integrated system" specific.

### How can this be tested?

1. visit the a basket page for a integrated system, like http://ue.odl.local:9080/cart/<integrated_system_slug>/
2. Add an item that is associated with the integrated system specified in step 1 to the basket.  Verify that the item shows that it was added to the basket.
3. Add an item that is NOT associated with the integrated system specified in step 1 to the basket.  Verify that the item does NOT show in the basket.
4. visit the a basket page for the integrated system associated with the item added to the basket in step 3, like http://ue.odl.local:9080/cart/<integrated_system_slug>/.  Verify that the item added to the basket in step 3 is displayed.
5. Click "checkout".  Verify that only the item added to the basket in step 3 is being charged.
6. Visit the URL from step 1.  The item added to the basket in step 2 should still be included in the basket.
7. Add an item that is NOT associated with the integrated system specified in step 1 to the basket.  Verify that the item does NOT show in the basket.
8. visit the a basket page for the integrated system associated with the item added to the basket in step 7, like http://ue.odl.local:9080/cart/<integrated_system_slug>/.
9. Press "clear cart".  Verify that the cart is cleared.
10. Visit the URL from step 1.  The item added to the basket in step 2 should still be included in the basket.
